### DISCUSSION TITLE
tree指定勾选策略 设置勾选策略来指定显示的勾选节点

### DISCUSSION THREAD

**Discussion by XiaolongMei** [element-plus/element-plus#17438] [Feature Request]
![image](https://github.com/element-plus/element-plus/assets/23044678/b3fd2cef-6f01-463b-a2e8-8b192669da40)
指定勾选策略
设置勾选策略来指定显示的勾选节点，all 表示显示全部选中节点；parent 表示只显示父节点（当父节点下所有子节点都选中时）；child 表示只显示子节点。

**Comment by nasirovx** [comment]
Yes, it is necessary to have this feature to allow for more flexible and customizable displays of selected nodes in a tree-select component.
